moped-toolbox/backfill_data_tracker_url_field/environment/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:focal
+RUN apt-get update
+RUN apt-get -y upgrade 
+RUN apt-get install -y aptitude vim procps
+RUN apt-get install -y python3 pip python3-psycopg2 black
+RUN pip install knackpy
+WORKDIR /root/app
+CMD ["tail", "-f", "/dev/null"]
```

Comment:
This seems nonsensical, but the effect is to tell the docker container to get started and then spin forever. The quick and dirty VM with libraries in place, essentially. 

If this script were ever going to be run with frequency, this CMD line would be changed, but this configuration makes firing the script up a very intentional process, you can't just `docker-compose up` and have it firing off changing operations.

Reply:
cool—TIL! so, how do you actually run the python command to run the script? docker-compose will bring up the container in interactive mode?

Reply:
ignore me! just read your instructions in the PR description

Reply:
btw, it's really nice to have a template here for getting `psycopg2` going quickly. always a headache to get the env set up

Reply:
I seems like those instructions in the PR description would be good to include in the README as a "Usage" section.